sql/core/src/main/resources/org/apache/spark/sql/execution/ui/static/spark-sql-viz.css

```diff
@@ -20,6 +20,12 @@
   text-shadow: none;
 }
 
+#plan-viz-graph svg g.cluster rect {
+  fill: #E5F2ff;
```

Comment:
Which one will you suggest? This is outer of operator, should be lighter than that, I choose this because it's between white and the one used for operators.